### ISSUE TITLE
Clean up history FB Community Guidelines 

### ISSUE THREAD

**Issue by pg-adrian** [OpenTermsArchive/pga-versions#1]
Most historical versions of the Facebook Community Guidelines just contain a change in links

**Comment by ckatzenbach** [comment]
Yes, that's need to part of "Re-Writing" the history. But that's on the agenda, right @afisher3578?
